### PR TITLE
Feature/header component

### LINKED ISSUE #6: MVP - Header section
As a stakeholder  
I want users to immediatly see the event's name and logo
In order to create conversions in the page to see its details and remaining sections

## ACCs
- [ ] Present the event name
- [ ] Present the event date with the format **Dec. 29**
- [ ] Have responsive behaviour both for mobile and desktop
  - [ ] Meet UI [specs](https://app.zeplin.io/project/5b3be80249cddedaa2fe54ba/screen/5b41f0ea0b64db87652d4acc)

screen shot 2018-07-08 at 22 22 10

### PR DESCRIPTION
**Pull request by Comum**
Closes #6

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# api endpoints
```
[the remark this answers is not in the document]
Comment (Comum) [reply docs/api.md]: 👍

File: src/scenes/Home.js [hunk 2]
```diff
@@ -1,21 +1,77 @@
-import React from 'react'
```
[the remark this answers is not in the document]
Comment (Comum) [reply src/scenes/Home.js]: 👍

File: src/scenes/Home.js [hunk 3]
```diff
@@ -1,21 +1,77 @@
...
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import PropTypes from 'prop-types'
 import Header from './components/Header/Header'
 import Body from './components/Body/Body'
 import Footer from './components/Footer/Footer'
+import { getOrganizationDetails } from '../store/actions/organization'
+import { getEventDetails } from '../store/actions/event'
 
 const HomeContainer = styled.div`
```
[the remark this answers is not in the document]
Comment (Comum) [reply src/scenes/Home.js]: 👍

File: src/scenes/components/Header/Header.js [hunk 4]
```diff
@@ -1,15 +1,62 @@
...
+const EventTitleSection = styled.section`
+  margin-bottom: 50px;
+`
+
+const Title = styled.section`
+  text-align: center;
+  margin-bottom: 30px;
+`
+
+const DateContaine = styled.section`
```
[the remark this answers is not in the document]
Comment (Comum) [reply src/scenes/components/Header/Header.js]: Forgot to change them

File: src/theme/fonts.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+// Font weights
```
[the remark this answers is not in the document]
Comment (Comum) [reply src/theme/fonts.js]: 👍

File: src/scenes/components/Header/Header.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (Comum) [reply src/scenes/components/Header/Header.js]: 👍

File: src/scenes/components/Header/Header.js [hunk 6]
```diff
@@ -1,15 +1,62 @@
...
+  </MainSection>
 )
 
+Header.defaultProps = {
+  date: '',
+}
+
+Header.propTypes = {
+  title: PropTypes.string.isRequired,
+  date: PropTypes.oneOfType([
```
[the remark this answers is not in the document]
Comment (Comum) [reply src/scenes/components/Header/Header.js]: 👍

File: src/store/reducers/event.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  FETCH_EVENTS_FULFILLED,
+  FETCH_EVENT_DETAILS_FULFILLED,
+} from '../actions/types'
+
+const INITIAL_STATE = {
+  events: [],
+  currentEvent: null,
+  date: '',
+  title: '',
+  event: {},
```
[the remark this answers is not in the document]
Comment (Comum) [reply src/store/reducers/event.js]: 👍

File: src/scenes/components/Header/Header.js [hunk 8]
```diff
@@ -1,15 +1,55 @@
...
+`
+
+const Header = props => (
+  <MainSection>
+    <LogoSection>
+      <img src="https://app.zeplin.io/img/icZeplin.svg" alt="logo" />
+    </LogoSection>
+    <EventTitleSection>
+      <Title>
+        <H3>{`${props.title}`}</H3>
```
Comment (OrencioRodolfo) [inline src/scenes/components/Header/Header.js]: No need for 
```js
{`${props.title}`}
```

It can be simply `{props.title}`

Comment (OrencioRodolfo) [reply src/scenes/components/Header/Header.js]: (or ever `{title}` if using destructuring at the function params) -> prefered version

File: src/scenes/HomeContainer.js [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+    this.props.getOrganizationDetails()
+    this.props.getEventDetails(0)
+  }
+
+  render() {
+    const { title, date } = this.props.event
+
+    return (
+      <HomeWrapper>
+        <Header title={title} date={date} />
```
Comment (OrencioRodolfo) [inline src/scenes/HomeContainer.js]: The header should also receive the WebMadeira logo by property (which comes from the store at organization details).

File: src/store/reducers/event.js [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+  FETCH_EVENTS_FULFILLED,
+  FETCH_EVENT_DETAILS_FULFILLED,
+} from '../actions/types'
+
+const INITIAL_STATE = {
+  events: [],
+  currentEvent: null,
+  date: '',
+  title: '',
+  eventId: {},
```
Comment (OrencioRodolfo) [inline src/store/reducers/event.js]: This must be a number.

File: src/scenes/HomeContainer.js [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  }
+
+  return {
+    ...state,
+    event: modifinedEvent,
+  }
+}
+
+function mapDispatchToProps(dispatch) {
```
[the remark this answers is not in the document]
Comment (Comum) [reply src/scenes/HomeContainer.js]: Didn't know this 👍

File: src/scenes/components/Header/Header.js (hunk 8 above)
Comment (Comum) [reply src/scenes/components/Header/Header.js]: 👍

File: src/scenes/HomeContainer.js (hunk 9 above)
Comment (Comum) [reply src/scenes/HomeContainer.js]: 👍

File: src/store/reducers/event.js (hunk 10 above)
Comment (Comum) [reply src/store/reducers/event.js]: 👍

**Pull request merged**